frontend/javascripts/admin/job/job_list_view.js

```diff
@@ -180,7 +180,7 @@ class JobListView extends React.PureComponent<Props, State> {
     ) {
       return (
         <span>
-          Applied merger mode to layer {job.layerName} of{" "}
+          Applying merger mode to layer {job.layerName} of{" "}
```

Comment:
why "applying"? because it's misleading if it's still processing? I find "Applied" better since it definitely works in the "done" state and also kinda works in the "in-progress" state.

Reply:
> because it's misleading if it's still processing?

Jep that was my thought.
> I find "Applied" better since it definitely works in the "done" state and also kinda works in the "in-progress" state.

sure, I got no strong opinion on this.